argo-cd-apps/base/all-clusters/infra-deployments/knative-eventing/knative-eventing.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I think, rather than `argo-cd-apps/base/all-clusters/` you want this in `argo-cd-apps/base/member/`.

While we still have the kubesaw architecture, there is a notion of `host` cluster and `member` cluster. The host clusters just provide proxying and user management. All the workload and user namespaces are on the member clusters. I think you only want knative eventing on the member clusters, just like you want kubearchive only on the member clusters.

Reply:
... eventually, we're going to consolidate and drop the `host` clusters at which point the only kind of cluster will be `member` clusters, but it's just not in place yet.

Reply:
Moved!